setup.py

```diff
@@ -2,10 +2,17 @@
 
 setup(
     name="imsms_analysis",
+    package_data={'imsms_analysis':
+        [
+            "dataset/*"
```

Comment:
Is this recursive?  I believe most of the files are in dataset/biom/* for example.  

This may dramatically increase installation time (I think one of the biom files I have locally is 12 GB).  Is it required to copy data around?  I would prefer to think of data as being separate from the code in this repository at least by design, even though for convenience we may include some right now.  May be out of scope for this PR, but I could add a separate dataset_location field to the config that would let us avoid this copy on installation right?

Reply:
I think it is recursive - haven't noticed a slowdown with install times though. Actually this is probably unnecessary so long as you always run `python whateveranalysis.py` from within `imsms_analysis/analyses/` and you have the right relative path to the data.